blocks/hero-block/_index.scss

```diff
@@ -0,0 +1,79 @@
+@use "@wpmedia/arc-themes-components/scss";
+
+.b-hero {
+	display: grid;
+	grid-template:
+		"hero"
+		"info";
+	justify-content: center;
+
+	@include scss.block-properties("hero");
+	@include scss.block-components("hero");
+
+	> * {
+		grid-area: hero;
+	}
+
+	&--image {
+		@include scss.block-properties("hero-image");
+		@include scss.block-components("hero-image");
+	}
+
+	&--text {
+		place-self: center;
+		max-width: 24rem;
+
+		@include scss.block-properties("hero-text");
+		@include scss.block-components("hero-text");
+	}
+
+	&--text.stacked {
```

Comment:
The `.stacked` selector should be scoped to the block `.b-hero--stacked` otherwise we will have issues if some other block uses a CSS class `stacked`

Reply:
Added the right conventions inline with your request